### DISCUSSION TITLE
Remove sending password reset requests via username

### DISCUSSION THREAD

**Discussion by umrua** [ppy/osu#15022] [Ideas] [answered]
Currently you are able to send password requests to any username in the game. Obviously, nothing can be done without someone having access to your email, but it's quite annoying seeing password requests showing up at random in the inbox. This has happened to me twice on different days. It'd be much better if it only allowed typing in the email.

This is my first time posting here so sorry if this is in the wrong place.

**Comment by gagahpangeran** [comment, marked as answer]
Please open an issue at https://github.com/ppy/osu-web/
